docs/time/performance.inc.rst

```diff
@@ -10,3 +10,52 @@ Performance Tips
 
 Here we provide some tips and tricks for how to optimize performance of code
 using `astropy.time`.
+
+Light Travel Times
+------------------
+
+Listed below are two approaches to calculating light travel times for tens of
+thousands of sources in a degree patch of the sky. The first approach calculates
+the travel times without iteration while the second calculates using iteration.
+
+
+>>> import numpy as np
+>>> import astropy.coordinates as coord
+>>> import astropy.units as u
+>>> from astropy.time import Time
+
+>>> ra = np.random.normal(0.0, 1.0, 50000)
+>>> dec = np.random.normal(0.0, 1.0, 50000)
+>>> coos = coord.SkyCoord(ra, dec, unit=u.deg)
+>>> observatory = coord.EarthLocation.from_geocentric(5327448.9957829, -1718665.73869569, 3051566.90295403, unit='m') #Lapalma Observatory
+
+Approach Not Using Iteration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>>> Time.light_travel_time(coos, location=observatory)
+
+   **CPU times:** user 56 ms, sys: 3.21 ms, total: 59.2 ms
+
+   **Wall time:** 58.2 ms
+
+Approach Using Iteration
+^^^^^^^^^^^^^^^^^^^^^^^^
+>>> ltts = [Time.light_travel_time(coo, location=coos.location) for coo in coos]
+
+   **CPU times:** user 16min 45s, sys: 5.08 s, total: 16min 50s
+
+   **Wall time:** 16min 58s
+
+As you can see, the user time is significantly longer.
+
+Avoid Iteration For Light Travel Time for Many Objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For user cases where there are thousands of times for each source, broadcasting
+can be used:
+
+>>> times = Time.now() + np.linspace(0, 3, 1000)*u.day
```

Comment:
I think we probably need to spell out this example quite a bit more. At the moment it's not quite clear what's going on to anyone without a detailed knowledge of numpy's broadcasting rules. 

A minimal change would be to separate out the step adding a new axis to ```coos```, show the shape afterwards and explain that the change of shape means that the 1000 times and the 50,000 co-ordinates will be broadcast against each other. You could possibly link to external information on Numpy broadcasting. 

At the end, I would show the shape of the resulting ```ltts``` object, since it will help clarify what has happened.